css/styles.css

```diff
@@ -0,0 +1,61 @@
+body {
+  background-color: rgb(240, 240, 240);
+}
+
+label,
+img {
+  display: block;
+}
+
+#purple-background {
+  background-color: rgb(86, 43, 129);
+  padding: 0;
+}
+
+#header-style {
+  color: white;
+  font-size: 1rem;
+  font-weight: normal;
+}
+
+#photo {
+  display: none;
+}
```

Comment:
This `id` doesn't exist, so this should be removed since it doesn't do anything. Remember to make sure all your CSS actually has an impact!